packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -586,153 +637,525 @@ async function handler(req: NextApiRequest & { userId?: number | undefined }) {
     seatsPerTimeSlot: eventType.seatsPerTimeSlot,
   };
 
-  // For seats, if the booking already exists then we want to add the new attendee to the existing booking
-  if (reqBody.bookingUid) {
+  const rescheduleUid = reqBody.rescheduleUid;
+
+  type BookingType = Prisma.PromiseReturnType<typeof getOriginalRescheduledBooking>;
+  let originalRescheduledBooking: BookingType = null;
+
+  if (rescheduleUid) {
+    originalRescheduledBooking = await getOriginalRescheduledBooking(
+      rescheduleUid,
+      !!eventType.seatsPerTimeSlot
+    );
+  }
+
+  if (!originalRescheduledBooking && rescheduleUid) {
+    throw new HttpError({ statusCode: 404, message: "Could not find original booking" });
+  }
+
+  /* Used for seats bookings to update evt object with video data */
+  const addVideoCallDataToEvt = (bookingReferences: BookingReference[]) => {
+    const videoCallReference = bookingReferences.find((reference) => reference.type.includes("_video"));
+
+    if (videoCallReference) {
+      evt.videoCallData = {
+        type: videoCallReference.type,
+        id: videoCallReference.meetingId,
+        password: videoCallReference?.meetingPassword,
+        url: videoCallReference.meetingUrl,
+      };
+    }
+  };
```

Comment:
If we're updating the original rescheduled booking calendar event we need to pass the video call data if there is any.